lib/cb/utils/validator.rb

```diff
@@ -65,7 +65,13 @@ def try_parse_xml(body)
       end
 
       def error_message(processed_response)
-        processed_response.fetch('errors', processed_response.fetch('Errors', ''))
+        return '' if processed_response.empty?
+        find_errors_node(processed_response) || ''
+      end
+
+      def find_errors_node(processed_response)
+        nested_hash = processed_response[processed_response.keys.first]
```

Comment:
I think we only want to look in the nested hash if there's a single top level key.